editor/src/components/canvas/ui-jsx-canvas.tsx

```diff
@@ -935,9 +935,7 @@ const SceneRoot: React.FunctionComponent<SceneRootProps> = (props) => {
   let rootElement = null
   let validPaths: Array<InstancePath> = []
   if (content != null) {
-    const defaultProps = defaultPropertiesForComponent(content)
```

Comment:
this is a pretty important change, I think the old behavior was explicitly broken since it didn't produce the same results as the Preview

Reply:
This would create different behaviour to the preview because the components wont have default props streamlined in.

Reply:
yeah, we are in agreement :) "old behavior was explicitly broken" what I meant was that it was a broken behavior, not that it was broken by this change :)